ddl/ddl.go

```diff
@@ -618,6 +785,11 @@ func (d *ddl) OwnerManager() owner.Manager {
 	return d.ownerManager
 }
 
+// SchemaVersionManager implements DDL.SchemaVersionManager interface.
+func (d *ddl) SchemaVersionManager() *SchemaVersionManager {
+	return d.ddlCtx.SchemaVersionManager
```

Comment:
It is better to use `d.ddlCtx.schemaVersionManager`, so that the "SchemaVersionManager" interface feels meaningful.